src/main/java/org/broadinstitute/hellbender/tools/copynumber/CollectAllelicCounts.java

```diff
@@ -28,22 +28,22 @@
 import java.util.List;
 
 /**
- * Collect reference and alternate allele counts at specified sites.  The alt count is defined as the
+ * Collects reference and alternate allele counts at specified sites. The alt count is defined as the
  * total count minus the ref count, and the alt nucleotide is defined as the non-ref base with the highest count,
- * with ties broken by the order of the bases in {@link AllelicCountCollector#BASES}.  Only reads that pass the
+ * with ties broken by the order of the bases in {@link AllelicCountCollector#BASES}. Only reads that pass the
  * specified read filters and bases that exceed the specified {@code minimum-base-quality} will be counted.
  *
- * <h3>Input</h3>
+ * <h3>Inputs</h3>
  *
  * <ul>
  *     <li>
- *         BAM file.
+ *         SAM format read data
  *     </li>
  *     <li>
- *         Reference file.
+ *         Reference FASTA file
  *     </li>
  *     <li>
- *         Sites at which allelic counts will be collected.
+ *         Sites at which allelic counts will be collected, e.g. sites of common germline variation
```

Comment:
Since we are putting this tool under CoverageAnalysisProgramGroup, you could imagine it might be used more generally to collect counts at sites.  No reason to point out sites of common germline variation specifically.